### PR TITLE
correct efsl syntax and remove extraneous quotemarks

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

EFSL had a syntax issue with a couple of URL references.  And, a quick scan of the specs found some extraneous quote marks with doc references, so I removed them.

### CODE REVIEW

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc [hunk 1]
```diff
@@ -678,27 +672,25 @@ resources.
 
 * The classes and resources accessible to any
 web modules included in the same ear file, as described in
-<<a3046, Web Container Class
-Loading Requirements>>” above.
+<<a3046, Web Container Class Loading Requirements>>” above.
```
Comment (NottyCode) [inline specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc:675]: did you mean to remove this quote?

File: specification/src/main/asciidoc/license-efsl.adoc [hunk 2]
```diff
@@ -34,7 +34,7 @@ document, or portions thereof, that you use:
 * All existing copyright notices, or if one does not exist, a notice
   (hypertext is preferred, but a textual representation is permitted)
   of the form: "Copyright (c) [$date-of-document]
-  Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php"
+  Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php[]"
```
Comment (NottyCode) [inline specification/src/main/asciidoc/license-efsl.adoc:37]: I thought that you would need link:https://www.eclipse.org/legal/efsl.php[]

File: specification/src/main/asciidoc/license-efsl.adoc [hunk 3]
```diff
@@ -55,7 +55,7 @@ The notice is:
 
 "Copyright (c) 2018,2020 Eclipse Foundation. This software or
 document includes material copied from or derived from 
-Jakarta EE Platform Specification, https://jakarta.ee/specifications/platform/9/."
+Jakarta EE Platform Specification, https://jakarta.ee/specifications/platform/9/[]."
```
Comment (NottyCode) [inline specification/src/main/asciidoc/license-efsl.adoc:58]: I thought that you would need link:https://jakarta.ee/specifications/platform/9/[]

File: specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc (hunk 1 above)
Comment (kwsutter) [reply specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc:675]: Yep!  :-) Thanks!

File: specification/src/main/asciidoc/license-efsl.adoc (hunk 2 above)
Comment (kwsutter) [reply specification/src/main/asciidoc/license-efsl.adoc:37]: Nope, the link: is not required.  Already verified the pdf and html generation.

File: specification/src/main/asciidoc/license-efsl.adoc (hunk 3 above)
Comment (kwsutter) [reply specification/src/main/asciidoc/license-efsl.adoc:58]: Nope.  The link: is not needed. Already verified.

**Pull request merged**